File: Swift-C++-Testsuite/test/functions/Swift/StaticFunctionReturnMultiParameter.swift
@main
struct MyMain {

    static func main() {
        let myRes = identity(x: 42, t: true)
    }
    static func identity(x: Int, t: Bool) -> Int {
        if t {
            return x
        } else {
            return -x
        }
    }

}

File: Swift-C++-Testsuite/test/functions/C++/StaticFunctionReturnMultiParameter.cpp
using namespace std;
 
 struct MyMain {
     static int identity(int x, bool t) {
         if (t) {
             return x;
         } else {
             return -x;
         }
     }
 };
 
 int main() {
         int myRes = MyMain::identity(42, true);
     return 0;
 }